tests/std/tests/VSO_0157762_feature_test_macros/test.compile.pass.cpp

```diff
@@ -411,6 +411,20 @@ STATIC_ASSERT(__cpp_lib_constexpr_complex == 201711L);
 #endif
 #endif
 
+#if _HAS_CXX20 && defined(__cpp_constexpr_dynamic_alloc)

```

Comment:
```suggestion
#if _HAS_CXX20 && defined(__cpp_constexpr_dynamic_alloc)  && (defined(__clang__) || defined(__EDG__))
```

@mnatsuhara should we add the constraints to the feature test macro as well since `yval_cores.h` will not define `__cpp_lib_constexpr_dynamic_alloc` except for Clang and EDG?

Reply:
Yeah, I realized a bit too late that I forgot to update this as well :)